releases/net471/dotnet471-known-issues.md

```diff
@@ -4,6 +4,7 @@
 This document lists the known issues that you may experience after you install the Microsoft .NET Framework 4.7.1.   
 
 ## Product issues for the .NET Framework 4.7.1   
+- [470006 - GC - Rare crashes can occur when background GC is enabled in applications which allocate and free large objects at a high rate](https://github.com/Microsoft/dotnet/blob/master/releases/net471/KnownIssues/470006-GC%20Crashes%20with%20high%20rate%20of%20large%20object%20allocation.md)
```

Comment:
which --> that

Reply:
Instead of "470006 - GC -" can we maybe say 470006 - **Runtime** -"?